src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheck.java

```diff
@@ -383,7 +400,7 @@ private static boolean shouldUpdateUninitializedVariables(DetailAST ast) {
      * @param containType the token type which has to be present in the branch
      * @return the matching token, or null if no match
      */
-    public DetailAST findLastChildWhichContainsSpecifiedToken(DetailAST ast, int childType,
+    public static DetailAST findLastChildWhichContainsSpecifiedToken(DetailAST ast, int childType,
```

Comment:
Since we are changing this line, please make it `private`. We shouldn't have any public methods in check except for overrides and setters unless for some very weird reason/hack.